client/src/instance-detail/instance-detail.js

```diff
@@ -13,6 +13,9 @@ import {createNode, deleteNode, getChildTypes} from '../tree/tree-util';
 import {values} from '../util/flow-util';
 import {getJson} from '../util/rest-util';
 import {getTypeNode, loadTypeNode} from '../util/node-util';
+import {LineChart, ColumnChart} from 'react-chartkick';
+import Chart from 'chart.js';
```

Comment:
I don't think this is being used, is it?  If not, kill it and remove it from the package.json